templates/components/members.html.twig

```diff
@@ -1,4 +1,4 @@
-<div class="component component--members u-full-width">
+<div class="component component--members u-full-width" data-anchor="no">
```

Comment:
@jean-gui The `data-anchor="no"` attribute should no longer be needed here, since the script has been modified to exclude headings with the `.visuallyhidden` class.